velox/exec/HashBuild.h

```diff
@@ -18,35 +18,103 @@
 #include "velox/exec/HashTable.h"
 #include "velox/exec/JoinBridge.h"
 #include "velox/exec/Operator.h"
+#include "velox/exec/Spill.h"
 #include "velox/exec/VectorHasher.h"
 #include "velox/expression/Expr.h"
 
 namespace facebook::velox::exec {
 
-// Hands over a hash table from a multi-threaded build pipeline to a
-// multi-threaded probe pipeline. This is owned by shared_ptr by all the build
-// and probe Operator instances concerned. Corresponds to the Presto concept of
-// the same name.
+/// Hands over a hash table from a multi-threaded build pipeline to a
+/// multi-threaded probe pipeline. This is owned by shared_ptr by all the build
+/// and probe Operator instances concerned. Corresponds to the Presto concept of
+/// the same name.
 class HashJoinBridge : public JoinBridge {
```

Comment:
This class grew quite a bit. Would it make sense to extract it into its own .h and .cpp files?